imagedephi/main.py

```diff
@@ -2,15 +2,18 @@
 
 import asyncio
 from pathlib import Path
+import pkgutil
+from typing import Optional, TextIO
```

Comment:
[`typing.Optional` is unnecessary](https://docs.python.org/3/library/typing.html#typing.Optional) (and unofficially deprecated) in Python 3.10. Just write `| None`.

Reply:
79fdcc4
Noted that `typing.Optional` is deprecated.